src/enzyme_ad/jax/Passes/Passes.td

```diff
@@ -20,6 +20,16 @@ def RemoveDuplicateFuncDefPass : Pass<"remove-duplicate-func-def", "mlir::Module
 
 def PropagateConstantBoundsPass : Pass<"propagate-constant-bounds", "ModuleOp"> {
   let summary = "Propagate constant bounds";
+  let description = [{
+    Propagate constant bounds information:
+    1. thread index 
+    2. Block dimension 
+    3. Block index.
+    Additionally, set all the following attributes for all the kernel pointers:
+    1. align 16
```

Comment:
technically I looked it up and we can assume 128 byte aligned: https://github.com/openxla/xla/blob/109a8ff382f2d174ee13504fb4f140a745cb25a2/xla/service/gpu/gpu_constants.h#L41